components/FMenu.vue

```diff
@@ -8,6 +8,7 @@
   Popper(
     :show="isOpen"
     placement="bottom-start"
+    strategy="fixed"
```

Comment:
I'm a little bit afraid by this change about unwanted side effects on our all our `FMenu` usages.
Have you tried?

Reply:
Yes I tried, I did not see any side effect (and I do not see any case where we would force to use `absolute` mode which is for me less convenient). Maybe I have missed some, indeed. Do you suggest to open that as a prop ?

Reply:
Yes why not, but maybe not with the `strategy` name, but maybe `position`. What do you think?

Reply:
Or, just we consider that we will use `fixed` 99% of the time, so it is this by default, and we add a boolean prop `absolute` to force it being absolute
```
  /**
   * Force the menu to be positioned as `absolute` instead of `fixed` by default
   */
  absolute?: boolean;
```
```suggestion
    :strategy="absolute ? 'absolute' : 'fixed'"
```

Reply:
I prefer to see a `<FMenu position="absolute" />` than a `<FMenu absolute />` that I find less informative. But as you like. :)

Reply:
I prefer `absolute`:
- more concise in usage
- as we toggle between "fixed" which is the default and "absolute", a boolean seems more suitable, instead of a prop which have two states but doing `position="fixed"` is exactly the same as doing nothing because of the default
- clear enough in the case of the menu, and if not:
- the prop is documented anyway ;-)

Reply:
we could have a boolean `absolute-position` if you think it's better